cjworkbench/i18n/trans.py

```diff
@@ -28,38 +28,35 @@ def _get_translations(locale):
     return _translators[locale]
 
 
-def trans(message_id, *, default, context=None, parameters={}):
-    """Translate the given message ID to the current locale
-    
-    HTML is not escaped.
+def trans(message_id, *, default, parameters={}):
+    """Mark a message for translation and localize it to the current locale, not escaping HTML.
     
     For code parsing reasons, respect the following order when passing keyword arguments:
-        `message_id` and then `default` and then `context` and then everything else
+        `message_id` and then `default` and then everything else
     """
     return _get_translations(get_language()).trans(
-        message_id, default=default, context=context, parameters=parameters
+        message_id, default=default, parameters=parameters
     )
 
 
-def trans_html(locale_id, message_id, *, default, context=None, parameters={}, tags={}):
-    """Translate the given message ID to the given locale
+trans_lazy = lazy(trans)
+"""Mark a string for translation, but actually localize it when it has to be used.
+   See the documentation of `trans` for more details on the function and its parameters.
+"""
+
+
+def localize_html(
+    locale_id, message_id, *, default, context=None, parameters={}, tags={}
```

Comment:
Is it right that we still have `context` here?

Reply:
It is. Due to different parsing algorithm, we can support context in template tags. I also test for it in the new extraction unit tests.